src/solvers/refinement/string_constraint.h

```diff
@@ -204,6 +208,21 @@ class string_not_contains_constraintt: public exprt
   }
 };
 
+inline static std::string from_expr(
+  const namespacet &ns,
+  const irep_idt &identifier,
+  const string_not_contains_constraintt &expr)
```

Comment:
My goal was to overload from expression in fact, but I couldn't see the proper way to do that. Where can I properly?